packages/admin/admin-date-time/src/datePicker/DatePicker.tsx

```diff
@@ -46,7 +47,7 @@ export const DatePicker = (inProps: DatePickerProps) => {
             value={value ? intl.formatDate(value, formatDateOptions) : ""}
             startAdornment={
                 <StartAdornment position="start" disablePointerEvents {...slotProps?.startAdornment}>
-                    <CalendarIcon />
+                    {startAdornment ? startAdornment : <CalendarIcon />}
                 </StartAdornment>
             }
```

Comment:
I think setting a `startAdornment` should have already been possible previously. 
It was included in `...inputWithPopperProps` and would have overridden this `startAdornment` prop. 

Now the possibly already passed in `startAdornment` would be wrapped in an additional (duplicate) `InputAdornment` component.